data/json/items/comestibles/cereal.json

```diff
@@ -0,0 +1,158 @@
+[
+  {
+    "abstract": "cereal_tpl",
+    "type": "COMESTIBLE",
+    "name": { "str_sp": "cereal" },
+    "weight": "40 g",
+    "volume": "250 ml",
+    "color": "yellow",
+    "symbol": "%",
+    "spoils_in": "90 days",
+    "container": "box_small",
+    "comestible_type": "FOOD",
+    "quench": -2,
+    "calories": 145,
+    "healthy": 1,
+    "description": "A generic box of cereal, you shouldn't see this.",
+    "price": 300,
+    "price_postapoc": 200,
+    "material": "wheat",
+    "flags": [ "EDIBLE_FROZEN", "USE_EAT_VERB" ],
+    "looks_like": "cereal",
+    "vitamins": [ [ "vitA", 10 ], [ "vitC", 7 ], [ "calcium", 7 ], [ "iron", 32 ] ],
+    "fun": 8
+  },
+  {
+    "abstract": "cereal_fpsugar_tpl",
+    "type": "COMESTIBLE",
+    "name": { "str_sp": "FoodPlace cereal" },
+    "quench": -3,
+    "calories": 225,
+    "healthy": -1,
+    "description": "A generic box of FoodPlace brand sugary cereal, you shouldn't see this.",
+    "price_postapoc": 400,
+    "charges": 6,
+    "material": "foodplace_foodstuff",
+    "looks_like": "cereal",
+    "vitamins": [ [ "vitA", 5 ], [ "vitC", 2 ], [ "calcium", 2 ], [ "iron", 16 ] ],
+    "fun": 11,
+    "addiction_potential": 5,
+    "addiction_type": "caffeine",
+    "fatigue_mod": 25,
+    "stim": 10,
+    "copy-from": "cereal_tpl"
+  },
+  {
+    "type": "COMESTIBLE",
+    "id": "cereal_fp1",
+    "name": { "str_sp": "Snicker-Snacks cereal" },
+    "description": "FoodPlace brand Snicker-Snack cereal.  Each tiny \"Snicker-Snack\" is shaped like an earth food!",
+    "material": "foodplace_foodstuff",
+    "charges": 6,
+    "copy-from": "cereal_tpl"
+  },
+  {
+    "type": "COMESTIBLE",
+    "id": "cereal_fp2",
+    "name": { "str_sp": "Carpenter Crunch cereal" },
+    "description": "This is FoodPlace brand \"Carpenter Crunch\" cereal with the iconic \"Breakfast Beaver\" mascot on the box.  It tastes kind of like nails.",
+    "material": "foodplace_foodstuff",
+    "charges": 6,
+    "vitamins": [ [ "vitA", 2 ], [ "vitC", 2 ], [ "calcium", 2 ], [ "iron", 50 ] ],
+    "fun": -1,
+    "spoils_in": "540 days",
+    "copy-from": "cereal_tpl"
+  },
+  {
+    "type": "COMESTIBLE",
+    "id": "cereal_fp3",
+    "name": { "str_sp": "Brantastic cereal" },
+    "description": "FoodPlace brand \"Brantastic\" cereal.  An essential part of a complete Bran breakfast.",
+    "material": "foodplace_foodstuff",
+    "charges": 6,
+    "copy-from": "cereal_tpl"
+  },
+  {
+    "type": "COMESTIBLE",
+    "id": "cereal_fp4",
+    "//note": "this is not a Fallout Brand, it's actually a calvin and hobbes fictional brand",
+    "name": { "str_sp": "Sugar Bombs cereal" },
+    "description": "FoodPlace brand \"Sugar Bombs\" cereal.  \"Chocolate Frosted Crunchy Sugar Bombs: 8 essential vitamins packed into that rich, fudgy taste!\"",
```

Comment:
Aren't Sugar Bombs straight out of Fallout? Would be better have a new name.

Reply:
Sounds good I'll change it.  My point was a bit suspect anyway - since they ripped it off Calvin and Hobbes we can too.